index.html

```diff
@@ -31,7 +31,7 @@
           <li><a href=""><img src="./img/video-camera.png" alt="" /></a></li>
           <li><a href=""><img class="avatar" src="https://source.unsplash.com/random" alt="" /></a>
           </li>
-          
+
         </ul>
 
       </div>
```

Comment:
Remove double spacing to increase readability

Reply:
- I believe the same suggestions stated in my first comments also go for this review 🌸

1. Using polite suggestions (" I think it will be better to ...... ") ✔️ 
2. Stating reasons why a requested change should be implemented ✔️ 
3. Adding emojis to make your review comments friendly✔️ 
4. Ask the student what they think. ✔️ 


_Why don't we edit your comments to meet all the requirements I have just stated above??_ 🤔